pkg/utils/common.go

```diff
@@ -85,6 +86,14 @@ func TargetRefToLatticeResourceName(
 	return "", fmt.Errorf("unsupported targetRef Kind: %s", targetRef.Kind)
 }
 
+func RandomString(length int) string {
+	randomSuffix := make([]rune, length)
+	for i := range randomSuffix {
+		randomSuffix[i] = rune(rand.Intn(26) + 'a')
+	}
+	return string(randomSuffix)
+}
+
```

Comment:
not generic random string, and should not be used in real code, please move into testing where you need it

Reply:
Will move to test code